Engine/src/Engine/ResourceManagement/Serializer.cpp

```diff
@@ -275,10 +276,11 @@ namespace Engine
 			{
 				
 				std::string texture = component["texName"];
+				std::string shader = component["shaderName"];
 				auto image = ResourceManager::getInstance()->getTexture(texture);
 				
 				//glm::vec4 color, GLuint texID, std::string texName, GLuint shaderID)
-				out.addComponent <MaterialComponent>(component["color"].get<glm::vec4>(), image.texID, texture, 0); //0 Would be shaderID. waiting until shader code is imported ********
+				out.addComponent <MaterialComponent>(component["color"].get<glm::vec4>(), image.texID, texture, shader); //0 Would be shaderID. waiting until shader code is imported ********
```

Comment:
remove the comment since shader code is done

Reply:
Done